### PR TITLE
fix isolated nodes in the dev environment

### PR DESCRIPTION
**Pull request by ryanpetrello**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -208,7 +208,7 @@ init:
 	if [ "$(AWX_GROUP_QUEUES)" == "tower,thepentagon" ]; then \
 		$(MANAGEMENT_COMMAND) provision_instance --hostname=isolated; \
 		$(MANAGEMENT_COMMAND) register_queue --queuename='thepentagon' --hostnames=isolated --controller=tower; \
-		$(MANAGEMENT_COMMAND) generate_isolated_key | ssh -o "StrictHostKeyChecking no" root@isolated 'cat >> /root/.ssh/authorized_keys'; \
+		$(MANAGEMENT_COMMAND) generate_isolated_key > /awx_devel/awx/main/expect/authorized_keys; \
```
Comment (ryanpetrello) [inline Makefile:211]: All of this key shuffling is complicated and doesn't work anymore w/ @shanemcd's recent `setuid` container changes.

File: tools/docker-isolated/Dockerfile [hunk 2]
```diff
@@ -23,8 +23,8 @@ RUN ssh-keygen -q -N "" -t dsa -f /etc/ssh/ssh_host_ecdsa_key
 RUN ssh-keygen -q -N "" -t rsa -f /etc/ssh/ssh_host_rsa_key
 RUN sed -i "s/#UsePrivilegeSeparation.*/UsePrivilegeSeparation no/g" /etc/ssh/sshd_config
 RUN sed -i "s/UsePAM.*/UsePAM yes/g" /etc/ssh/sshd_config
-RUN ssh-keygen -A
```
Comment (ryanpetrello) [inline tools/docker-isolated/Dockerfile:26]: This PR sets up a symlink for the isolated key.

The `awx_tools_1` container generates and writes a key at `/awx_devel/awx/main/expect/`, which is mapped to the shared volume for the source code.

The `awx_isolated_1` container sets up a symlink to point `/root/.ssh/authorized_keys` at the synced public keyfile we generate.

File: Makefile [hunk 3]
```diff
@@ -548,11 +548,6 @@ docker-isolated:
 	docker start tools_awx_1
```
Comment (AlanCoding) [inline Makefile:548]: you didn't get an error that the create command was deprecated? This command didn't work for me when I tried it.

but the rest of these changes look good.

Comment (ryanpetrello) [reply Makefile:548]: I'm using the latest `docker-ce` on Fedora and this seems to work for me:

```
docker --version
Docker version 18.09.0, build 4d60db4
```

Fresh `make docker-isolated` for me:

image

**Review by AlanCoding — APPROVED** [review]

**Pull request merged**